### ISSUE TITLE
Underline characters in preformatted block don't show up

### ISSUE THREAD

**Issue by t-900-a** [skyjake/lagrange#204]
Trying to get some ascii displaying correctly.
It displays correctly in amfora cli, but not lagrange.

https://github.com/t-900-a/gemmit/commit/f3ff083678e9e51b81623ffefd0e78086fb1de70

ascii art should look like this

https://camo.githubusercontent.com/9531b4b74cc00041f19eaa5353ad2a3dc774e62ecb3e7785dad83d9d449b1139/68747470733a2f2f736961736b792e6e65742f41414232304c536b47794e4e566c7175333533566452456333355144505f786d4773735a6c454a4f464b4b4a6541

thank you, lagrange is wonderful

**Comment by skyjake** [comment]
Gemtext ASCII art should be enclosed in a preformatted block, just like in Markdown.

`````
```
  .     '     ,
    __GEMMIT__
 _ /_|_____|_\ _
   '. \   / .'
     '.\ /.'
       '.'
```       
`````

**Comment by skyjake** [comment]
I noticed that you're using underlines in the GEMMIT part, and these do not show up in Lagrange's preformatted block for some reason. I'll take a look.

**Comment by skyjake** [comment]
This is likely caused by the lack of support for glyph compositing.

**Comment by skyjake** [comment]
Fixed in v1.6 (when using HarfBuzz).
